v4/weaviate/graphql/aggregatequerybuilder.go

```diff
@@ -0,0 +1,117 @@
+package graphql
```

Comment:
I would leave the name `aggregate.go` I know that you wanted to align this change with `getquerybuilder.go` but for get we have `get.go` that uses `getquerybuilder.go` and that's why we have the name for this like `getquerybuilder.go` I think that we can leave the `aggregate.go`